config/ols-config/ols-config.yaml

```diff
@@ -79,6 +79,22 @@ ontologies:
 #    label_property:  http://www.w3.org/2004/02/skos/core#prefLabel
     oboSlims: false
     ontology_purl : file:/mnt/ontologies/hp-branch-lymphoma.owl
+  - id: vbodev
```

Comment:
nit: shouldn't this be `vbo` following the id pattern for other "in development" ontologies?

Reply:
```suggestion
  - id: vbo
```